tsml_eval/_wip/shapelets/classification/shapelet_based/_stc.py

```diff
@@ -15,7 +15,7 @@
 from aeon.base._base import _clone_estimator
 from aeon.classification.base import BaseClassifier
 from aeon.classification.sklearn import RotationForestClassifier
-from tsml_eval._wip.shapelets.transformations.collection.shapelet_based import RandomShapeletTransform
+from tsml_eval._wip.shapelets.transformations.collection.shapelet_based._shapelet_transform import RandomShapeletTransform
```

Comment:
how can i fix this to not have to specify the python file of the class? modifying __init__ didn't seem to work for me